daily-challenges/2024-09/14/hunghoang3110.spec.ts

```
@@ -0,0 +1,45 @@
+// Javascript
+function calculateBMI(weight, height) {
+    const BMI_THRESHOLD = weight / (height * height)
+    if (weight <= 0 || height <= 0) {
+        throw new Error('Weight and height must be positive numbers');
+    }
+    const BMI_CLASSIFICATIONS = {
+        UNDERWEIGHT: { threshold: 18.5, message: 'Underweight' },
+        NORMAL: { threshold: 24.9, message: 'Normal' },
+        OVERWEIGHT: { threshold: 29.9, message: 'Overweight' },
+        OBESE: { threshold: Infinity, message: 'Obese' }
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)